getting_started.md

```diff
@@ -278,10 +278,12 @@ you're assigned a small trial project. Let the software team know in case your
 user needs to have access to an existing billing project or if you need to
 create a new billing project.
 
-At the moment, you can submit jobs to Hail Batch by running the "driver" program
-(which defines the batch) locally. However, since that's problematic in terms of
-reproducibility, we're currently looking into ways to run the driver itself on
-Hail Batch, too.
+You can submit jobs to Hail Batch by running the "driver" program (which
+defines the batch) locally, which is handy for prototyping and testing.
+However, since that's problematic in terms of
+[reproducibility](reproducible_analyses.md) for analyses on production data,
```

Comment:
I think giving more context for why analysis runner here might be useful, eg:

```suggestion
[reproducibility](reproducible_analyses.md) for analyses on production data due to the job graph not being persisted, you can use the [analysis runner](https://github.com/populationgenomics/analysis-runner) which runs a batch pipeline from a specific commit from a GH repository to run the driver itself on Hail batch.
```

Reply:
The graph actually does get persisted (it's essentially the "job spec" in a GCP bucket), but that's not really useful in practice to re-run something a year later ;). I've extended the sentence slightly based on your suggestion, but I don't want to copy too much of the explanation in `reproducible_analyses.md`.